subscription_events.go

```diff
@@ -53,7 +53,7 @@ type SubscriptionEventParams struct {
 	Params *SubscriptionEvent `json:"subscription_event"`
 }
 
-func (api API) ListSubscriptionEvents(filters *FilterSubscriptionEvents, cursor *MetaCursor) (*SubscriptionEvents, error) {
+func (api API) ListSubscriptionEvents(filters *FilterSubscriptionEvents, cursor *Cursor) (*SubscriptionEvents, error) {
```

Comment:
This won't let us run a cursored query that also has pagination filters like `per_page: 1` right? 

```go
type FilterSubscriptionEvents struct {
	CustomerExternalID     string `json:"customer_external_id,omitempty"`
	DataSourceUUID         string `json:"data_source_uuid,omitempty"`
	EffectiveDate          string `json:"effective_date,omitempty"`
	EventDate              string `json:"event_date,omitempty"`
	EventType              string `json:"event_type,omitempty"`
	ExternalID             string `json:"external_id,omitempty"`
	PlanExternalID         string `json:"plan_external_id,omitempty"`
	SubscriptionExternalID string `json:"subscription_external_id,omitempty"`
}
```